hpa/stormforger/sf-experiment/experiment.yaml

```diff
@@ -0,0 +1,225 @@
+apiVersion: redskyops.dev/v1beta1
+kind: Experiment
+metadata:
+  labels:
+    redskyops.dev/application: hpa-sf
+    redskyops.dev/scenario: standard
+  name: hpa-sf
+spec:
+  metrics:
```

Comment:
Might want to add this to the readme since we've got >2 metrics. 

```
In order to run this experiment you must have a paid version of StormForge Optimize. To upgrade visit our [pricing page](https://www.stormforge.io/pricing/).
```